core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -157,16 +156,28 @@ oppia.controller('FeedbackTab', [
           },
           newContent: function() {
             return $scope.activeThread.getReplacementHtmlFromSuggestion();
+          },
+          stateName: function() {
+            return $scope.activeThread.suggestion.stateName;
+          },
+          suggestionId: function() {
+            return $scope.activeThread.suggestion.suggestionId;
```

Comment:
Can we use the functions defined in SuggestionObjectFactory or SuggestionThreadObjectFactory instead of directly accessing the values.